src/MuTalk-Model/MethodMutation.class.st

```diff
@@ -145,6 +154,14 @@ MethodMutation >> runTests [
 		  runStoppingOnFirstFailOrError
 ]
 
+{ #category : 'private' }
+MethodMutation >> runTestsWithoutStoppingOnError [
+
+	^ (TestSuite named: 'Tests cases for: ' , self printString)
+		  addTests: testCaseReferences;
+		  run
+]
```

Comment:
Maybe the boolean should arrive here instead! We should have only one method `runTestsStoppingOnError: aBoolean` and that method will use the if correctly without repeating all the code!